.dockerignore

```diff
@@ -1,2 +1,4 @@
 docker-machine*
-.git
```

Comment:
Are you concerned about speed? If so, it should not impact perf in any significant way.